lib/db/memory.js

```diff
@@ -242,6 +242,18 @@ MemoryStore.prototype = {
     delete this.tokens[unbuf(id)];
     return P.resolve();
   },
+  /**
+   * Get all services that have have non-expired tokens
+   * @param email User's email
+   * @returns {*}
+   */
+  getActiveServicesByEmail: function getActiveServicesByEmail(email) {
```

Comment:
In particular, @seanmonstar, I have a feeling we were trying to move away from using email addresses in this codebase, and instead operating using only uids and looking up emails in the auth-server when necessary.  It's been a while, am I remembering that correctly?